### DISCUSSION TITLE
Issues converted to Discussions should be closed as not planned instead of as completed

### DISCUSSION THREAD

**Discussion by ouuan** [community/community#43497] [Discussions]
### Select Topic Area

Product Feedback

### Body

Now when an issue is converted to a discussion, it is closed as completed (purple), but it should be closed as not planned (gray).

**Comment by Animeshsingh17** [comment]
To address the issue of closing an issue as "not planned" (gray), you can follow these steps manually:
Close the Issue: As usual, close the issue by marking it as "completed" (purple).

1. Add a Comment: Leave a comment on the closed issue explaining that it has been moved to GitHub Discussions as a "not planned" discussion. This comment will help maintain the context for users who might come across the closed issue.
2. Create a Discussion: Create a new discussion in GitHub Discussions, copying the relevant content from the closed issue. In the new discussion, you can categorize it as "not planned" or add any relevant tags to indicate its status.
3. Reference the Discussion: In the closed issue's comment section, provide a link to the newly created discussion in GitHub Discussions. This way, users can easily find the continuation of the conversation in the discussion space.

**Reply by ouuan** [reply]
I certainly can do all these steps manually. But now if I want to convert an issue to a discussion it can be done in just two clicks. I don't want these many steps just for the gray close.
